submissions/kovalov/html-css-popup/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Kottans | Popup</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__container">
+        <nav class="navigation">
+          <ul class="navigation__list">
+            <li class="navigation__list-item">
+              <a href="#" class="header__title navigation__link">
```

Comment:
Logically this element looks like a page logo and not like part of page navigation, if you want to add a logo just remove him from navigation.